### PR TITLE
Add disclaimer block for articles containing affiliate links

### PR DESCRIPTION
**Pull request by rcrphillips**
## What does this change?
Adds the disclaimer block for when we have an article that contains affiliate links. The font-size is unfortunately magic number, but there's no existing size within the design system to enable it otherwise.

### Frontend
Screen Shot 2020-07-29 at 16 39 41

### DCR
Screen Shot 2020-07-29 at 16 39 23

## Why?
https://trello.com/c/yI6FAg1H/1679-support-affiliate-links

### CODE REVIEW

File: src/web/components/elements/DisclaimerBlockComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    sup {
+        font-size: 12.75px;
+    }
+`;
+
+export const DisclaimerBlockComponent: React.FC<{
+    html: string;
+    pillar: Pillar;
+}> = ({ html, pillar }) => (
+    <div
```
Comment (jfsoul) [inline src/web/components/elements/DisclaimerBlockComponent.tsx:22]: I'm wondering whether there's a better choice for semantic markup rather than div...

Comment (rcrphillips) [reply src/web/components/elements/DisclaimerBlockComponent.tsx:22]: Uses <span> in frontend, but it felt easy to use <div> to ensure padding is properly set.

File: src/web/components/elements/DisclaimerBlockComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+const style = (pillar: Pillar) => css`
+    ${body.small()};
+
+    a {
+        color: ${pillarPalette[pillar].dark};
+    }
+
+    sup {
+        font-size: 12.75px;
```
Comment (jfsoul) [inline src/web/components/elements/DisclaimerBlockComponent.tsx:14]: Frontend is `font-size: 75%` which might get you the same but a little less magic.

Comment (rcrphillips) [reply src/web/components/elements/DisclaimerBlockComponent.tsx]: Have resolved.

File: src/web/components/elements/DisclaimerBlockComponent.tsx (hunk 1 above)
Comment (rcrphillips) [reply src/web/components/elements/DisclaimerBlockComponent.tsx]: Have altered back to span.

Comment (jfsoul) [reply src/web/components/elements/DisclaimerBlockComponent.tsx]: I couldn't see any wrapping element on frontend, but I think we have to have one here to style the `html`. In which case maybe a `footer` is a reasonable choice?

>The HTML footer element represents a footer for its nearest sectioning content or sectioning root element. A footer typically contains information about the author of the section, copyright data or links to related documents.

**Pull request merged**